src/components/Editor/TextEditor.tsx

```diff
@@ -36,10 +37,12 @@ const TextEditor = (): JSX.Element => {
   return (
     <Rnd
       cancel=".noDrag"
-      minWidth="200"
-      minHeight="150"
-      maxWidth="400"
-      maxHeight="250"
+      minWidth="200px"
```

Comment:
A possible enhancement could be changing the default text box size. It can be a little increased perhaps so that it's more visible when selected. So basically, something like this:
 minWidth="250px"
 minHeight="170px"

Reply:
Sure will fix this